app/scripts/metamask-controller.js

```diff
@@ -1014,7 +1014,7 @@ export default class MetamaskController extends EventEmitter {
     this.keyringController = new KeyringController({
       keyringBuilders: additionalKeyrings,
       state: initState.KeyringController,
-      encryptor: opts.encryptor || undefined,
+      encryptor: opts.encryptor || encryptorFactory(600_000),
```

Comment:
I was tempted, but these were my considerations:
- `opts.encryptor` is not documented anywhere, and if we pass it from `background.js` we also lose information coming from `KeyringController` jsdocs
- We now have two encryptors (one used for the keyring controller, and one for snaps), so passing the encryptor from `background.js` might be confusing as it would be used only for KeyringController

Reply:
Actually, I was considering removing `opts.encryptor` completely, but I'm not sure that is _really_ never used

Reply:
Oh, of course, it is used for test purposes - we can't use a browser encryptor in unit tests.